ReleaseHistory.md

```diff
@@ -18,6 +18,10 @@
 - UER => Eliminate unhandled exceptions in rule.
 - UEE => Eliminate unhandled exceptions in engine.
 
+## v4.5.0 UNRELEASED
```

Comment:
>UNRELEASED [](http://example.com/codeflow?start=10&length=10)

This is not a draft PR but it is an intermediate, exploratory sort of check-in. i.e., this change attempts to use the new SARIF SDK event collection mechanism but also demonstrates how to add a custom event (in our case, the 'sniff' functionality). I'd like to review/refine this a bit, and then we can discuss a general test methodology. Ideally, this methodology would exist in the SARIF SDK however.